langchain/schema.py

```diff
@@ -646,3 +646,49 @@ def save(self, file_path: Union[Path, str]) -> None:
                 yaml.dump(prompt_dict, f, default_flow_style=False)
         else:
             raise ValueError(f"{save_path} must be json or yaml")
+
+
+def format_document(doc: Document, prompt: BasePromptTemplate) -> str:
```

Comment:
for sep pr but what if we had
```python
class DocumentPromptTemplate(PromptTemplate):
  doc_content_key: str = "page_content"
  def format(self, document: Optional[Document]=None, **kwargs: Any) -> str:
    if document:
      kwargs[self.doc_content_key] = doc.page_content
      kwargs.update(doc.metadata)
    return super().format(**kwargs)
```

and then you could easily make LLMChains that take docs